pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -229,6 +229,13 @@ func (r *ReconcileComplianceSuite) issueValidationError(suite *compv1alpha1.Comp
 }
 
 func (r *ReconcileComplianceSuite) reconcileScans(suite *compv1alpha1.ComplianceSuite, logger logr.Logger) (bool, error) {
+	// Short-circuit and return without requeuing the request if the
+	// ComplianceSuite is suspended by the ScanSetting. When the
+	// ScanSetting.suspend attribute is disabled, or set to False, the
+	// ComplianceSuite will get updated and schedule scans.
+	if suite.Spec.Suspend {
+		return false, nil
```

Comment:
I got down the path of incorporating status into the ComplianceSuite, and realized that having a Suspended status there makes reconciling it when the `ScanSetting` is reactivated much more complicated.

I'll propose an alternative with the status on the `ScanSettingBinding`, which should allow us to relay the same information, but it makes it simpler to reconcile because we can short-circuit the `ComplianceScan` creation if we have a binding that references a suspended scan setting.